src/test/java/core/basesyntax/dao/StorageDaoImplTest.java

```diff
@@ -0,0 +1,122 @@
+package core.basesyntax.dao;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class StorageDaoImplTest {
+    private static final int LOWEST_VALID_AGE = 18;
+    private static final int ADULT_AGE = 32;
+    private static final String LOWEST_VALID_PASS = "qw21re";
+    private static User validUser1;
+    private static User validUser2;
+    private static User validUser3;
```

Comment:
```suggestion
    private static User firstValidUser;
    private static User secondValidUser;
    private static User thirdValidUser;
```
the same for the next class